Reload scripts (#9977)
withastro/astro: .changeset/rare-coins-jump.md

---
"astro": minor
---

> Add option to add `data-astro-reload` attribute to scripts so that they will be re-executed after transition

Reviewer: ```suggestion
Add option to add `data-astro-reload` attribute to scripts so that they will be re-executed after view transitions
```

> Add option to add `data-astro-reload` attribute to scripts so that they will be re-executed after view transitions

Reviewer 2: ```suggestion
Supports adding the `data-astro-rerun` attribute on script tags so that they will be re-executed after view transitions
```
